File: Examples/Features/Formatting/Fonts/FontColor.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Formatting.Fonts
{
    public class FontColor : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];
            worksheet.Range["A1"].Value = "font";
            worksheet.Range["A1"].Font.Color = Color.Green;
        }
    }
}

File: Examples.VB/Features/Formatting/Fonts/FontColor.vb
Imports System.Drawing

Namespace Features.Formatting.Fonts
    Public Class FontColor
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)
            worksheet.Range!A1.Value = "font"
            worksheet.Range!A1.Font.Color = Color.Green
        End Sub
    End Class
End Namespace
